<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>labels</class>
 <widget class="QDockWidget" name="labels">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>275</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFeatureMask</set>
  </property>
  <property name="windowTitle">
   <string>Labels</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTreeView" name="lvLabels">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="5" column="0" colspan="2">
       <widget class="QCheckBox" name="cbEnableEditing">
        <property name="text">
         <string>Enable Editing</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QSlider" name="slOpacity">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbRadius">
        <property name="text">
         <string>Radius:</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="2">
       <widget class="QPushButton" name="pbStoreImage">
        <property name="text">
         <string>Store Image</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="layoutButtons">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pbAdd">
          <property name="toolTip">
           <string>Add label</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../libs/guibase/guibase.qrc">
            <normaloff>:/images/plus.svg</normaloff>:/images/plus.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbRemove">
          <property name="toolTip">
           <string>Remove selected label</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../libs/guibase/guibase.qrc">
            <normaloff>:/images/minus.svg</normaloff>:/images/minus.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lbColorTheme">
        <property name="text">
         <string>Color Theme:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QPushButton" name="pbSample">
        <property name="text">
         <string>Sample</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="cbColorTheme"/>
      </item>
      <item row="13" column="0" colspan="2">
       <widget class="QPushButton" name="pbStore">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Store</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lbOpacity">
        <property name="text">
         <string>Opacity:</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0" colspan="2">
       <widget class="QPushButton" name="pbLoad">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QPushButton" name="pbClear">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QCheckBox" name="cbTrackSeeds">
        <property name="text">
         <string>Track Seeds</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lvLabels</tabstop>
  <tabstop>pbAdd</tabstop>
  <tabstop>pbRemove</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>cbEnableEditing</tabstop>
  <tabstop>cbColorTheme</tabstop>
  <tabstop>slOpacity</tabstop>
  <tabstop>pbClear</tabstop>
  <tabstop>pbSample</tabstop>
  <tabstop>pbStoreImage</tabstop>
  <tabstop>pbStore</tabstop>
  <tabstop>pbLoad</tabstop>
 </tabstops>
 <resources>
  <include location="../../libs/guibase/guibase.qrc"/>
 </resources>
 <connections/>
</ui>
